hasher-matcher-actioner/hmalib/metrics/__init__.py

```diff
@@ -75,7 +75,7 @@ def _no_op_timer(name):
     yield
 
 
-def _no_op_flush():
+def _no_op_flush(namespace: str = "does not matter"):
```

Comment:
nit: ` = "no_po"` is it possible spaces in the namespace might mess something up downstream?

Reply:
Extremely unlikely. This is just to get mypy to stop crying. This string is not published anywhere.